hunts/src/HuntViewMain.js

```diff
@@ -169,28 +169,36 @@ export const HuntViewMain = (props) => {
             <input
               style={{ margin: "0 5px 0 10px" }}
               type="radio"
-              checked={filterSolved === SOLVE_STATE_FILTER_OPTIONS.PRIORITY}
+              checked={
+                filterSolved === SOLVE_STATE_FILTER_OPTIONS.PRIORITY ||
+                filterSolved === SOLVE_STATE_FILTER_OPTIONS.UNSOLVED
+              }
               onChange={(evt) => {
                 if (evt.target.checked) {
                   setFilterSolved(SOLVE_STATE_FILTER_OPTIONS.PRIORITY);
                 }
               }}
             ></input>
-            Priority
-          </label>
-          <label>
-            <input
-              style={{ margin: "0 5px 0 10px" }}
-              type="radio"
-              checked={filterSolved === SOLVE_STATE_FILTER_OPTIONS.UNSOLVED}
-              onChange={(evt) => {
-                if (evt.target.checked) {
-                  setFilterSolved(SOLVE_STATE_FILTER_OPTIONS.UNSOLVED);
-                }
-              }}
-            ></input>
             Unsolved
           </label>
+          {(filterSolved === SOLVE_STATE_FILTER_OPTIONS.PRIORITY ||
+            filterSolved === SOLVE_STATE_FILTER_OPTIONS.UNSOLVED) && (
+            <label>
+              <input
+                style={{ margin: "0 5px 0 10px" }}
+                type="checkbox"
+                checked={filterSolved === SOLVE_STATE_FILTER_OPTIONS.UNSOLVED}
```

Comment:
Do you mind renaming PRIORITY->UNSOLVED and UNSOLVED->UNSOLVED_WITH_SOLVED_METAS or something equivalent? (also add comments with more details on what they mean, perhaps with an example)